app/src/main/java/org/openobservatory/ooniprobe/receiver/TestRunBroadRequestReceiver.java

```diff
@@ -56,7 +56,7 @@ public void onReceive(Context context, Intent intent) {
                 listener.onRun(value);
                 break;
             case TestAsyncTask.PRG:
-                if (service != null) {
+                if (service != null && service.task.testSuites.indexOf(service.task.currentSuite) > 0) {
```

Comment:
I am in doubt whether this is a fix to avoid accessing an empty suite or whether this is some other kind of fix, so it would help my understanding if you could explain why there's this change here! 🙂 🙏

Reply:
There is a possibility that `service.task.currentSuite`  does not exist in `service.task.testSuites`